### PR TITLE
Adds unity profile config/docs

### PR DESCRIPTION
**Pull request by baldikacti**
---
name: New Config
about: A new cluster config
---

Please follow these steps before submitting your PR:

- [ ] If your PR is a work in progress, include `[WIP]` in its title
- [x] Your PR targets the `master` branch
- [ ] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [x] Add your custom config file to the `conf/` directory
- [x] Add your documentation file to the `docs/` directory
- [x] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [x] Add your custom profile to the `README.md` file in the top-level directory
- [x] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`
- [ ] OPTIONAL: Add your custom profile path and GitHub user name to `.github/CODEOWNERS` (`**/<custom-profile>** @<github-username>`)

### CODE REVIEW

File: docs/unity.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+```bash
+## Load Nextflow and Apptainer environment modules
+module purge
+module load nextflow/24.04.3
+module load apptainer/latest
+
+nextflow run <pipeline_name> -profile unity
+```
+
+Apptainer module will look for a cache directory with the name `.apptainer/cache` in your `/home` or `/work` directory. Please create the cache directory in `/work/pi_<pi_name>/.apptainer/cache` or `/work/pi_<pi_name>/<username>/.apptainer/cache` to prevent apptainer filling your HOME directory.
```
Comment (jfy133) [inline docs/unity.md:20]: You could also use the NXF_SINGULARITY_CACHEDIR and NXF_SINGULARITY_LIBRARYDIR variables to point to this instead.

File: docs/unity.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## iGenome database
+
+A local copy of the iGenomes resource has been made available on UNITY CLUSTER so you should be able to run the pipeline against any reference available in the `igenomes.config` specific to the nf-core pipeline.
```
Comment (jfy133) [inline docs/unity.md:24]:
```suggestion
A local copy of the iGenomes resource has been made available on UNITY CLUSTER so you should be able to run a pipeline supporting this against any reference available in the `igenomes.config` specific to the nf-core pipeline.
```

**Review by jfy133 — APPROVED** [review]

**Comment by jfy133** [comment]
@baldikacti you can merge whenever you are ready. If you don't have the option ping me and I can do it for you

**Comment by baldikacti** [comment]
@jfy133, thank you for the comments. Yes, would you mind merging it, please? I do not see the option.

**Comment by jfy133** [comment]
Done @baldikacti !

**Pull request merged**